src/main/resources/CodyBundle.properties

```diff
@@ -126,9 +126,51 @@ PromptPanel.ask-cody.message=Message (type @ to include specific files as contex
 PromptPanel.ask-cody.follow-up-message=Follow-up message (type @ to include specific files as context)
 LlmDropdown.disabled.text=Start a new chat to change the model
 
-# Chat actions
+# Actions Groups
```

Comment:
Juts one comment, pleas make sure it works in all context menus, actions when searched using `shift shift`, etc

Reply:
I double checked that - works fine